### ISSUE TITLE
Bungee issue

### ISSUE THREAD

**Issue by AlonsoAliaga** [MrMicky-FR/FastBoard#4]
I'm using this for a plugin, but there have been reports that players cannot login to the server when this is used in spigot server under a bungeecord (I mean, using bungeecord) As soon as the scoreboard is assigned, they are kicked from the server.

Any other report related to a issue similar to this one?

**Comment by MrMicky-FR** [comment]
It's the first time I have an issue like this and I can't reproduce it on a local test server with BungeeCord, wich Spigot version you use, wich BungeeCord version you use and wich version the player use ? Also what message has the player when he is kicked ?

**Comment by AlonsoAliaga** [comment]
Hello, The proxy is currently using these plugins:
![image](https://i.gyazo.com/2e8c6e8de299a57f483fc7c08eae0864.png)
The spigot server uses these plugins:
![image](https://user-images.githubusercontent.com/36780081/95027572-32ddf980-065f-11eb-9ea2-5badd93a1dbf.png)
Current proxy version
![image](https://user-images.githubusercontent.com/36780081/95027593-5dc84d80-065f-11eb-8760-30e19872228e.png)
Current spigot version
![image](https://user-images.githubusercontent.com/36780081/95027603-70428700-065f-11eb-8cce-a1080745ef16.png)
Plugin that breaks and kick on join is mine, this is the link
https://www.spigotmc.org/resources/84049/
I've tested without bungeecord and works fine, tested wit bungeecord without scoreboard (disabled in config) and works fine,
issue is with bungeecord and scoreboard, could you please check it out?
Kick message is the usual "The server you were previously shut down, you were moved to callback server"
but i need to clarify server doesnt shut down, its just the message so it's probably due to other plugin sending that message when you are kicked from a server.

**Comment by MrMicky-FR** [comment]
Thanks for this information I will look at this soon, just with wich Minecraft client version you have this issue ?

**Comment by AlonsoAliaga** [comment]
I'm currently using Badlion Client 1.16.3
Other player with Labymod 1.12.2 also reports being kicked
we are using ViaVersion, might that be a problem? or maybe TAB plugin?
Could you check it please? Thanks :D

I'm looking forward to use this utils since working with bukkit-api scoreboard is a bit complicated,
this is a perfect solution but i need to fix that first, thanks in advance

**Comment by MrMicky-FR** [comment]
I tested with Travertine with ViaVersion/ViaBackwards on Paper 1.11.2 with the tab plugin and the scoreboard is working without any issue.

Currently I don't have access to my McMarket account to download FlameCord and I don't have the time to download 50 plugins to try to reproduce the exactly same setup, could you please try to replace FlameCord with Travertine and if you still have this issue, try to reproduce with only your plugin so we can find what's the cause of this issue (maybe it's a plugin or maybe it's something else).
Thanks

**Comment by MrMicky-FR** [comment]
Also do you have any error in the console (On Bungee or Spigot) ?

**Comment by AlonsoAliaga** [comment]
Issue is from a network that is using my plugin so I can't change proxy, as long as i could see there isn't any error in bungeecord console.
However in spigot console appears `Terryy lost connection: Internal Exception: java.io.IOException: Error while read(...): Connection reset by peer` but nothing else. Im going to make some tests too in my localhost later with current plugins and I'll let you know

**Comment by AlonsoAliaga** [comment]
Hello, its been 2 days. I finally figure out what was the issue. It was from my side, to make sure scoreboards are displayed correctly (in case another scoreboard overrides the one i created or to make it appear if somehow it was removed) i added a new method that re-sends the scoreboard, however, since i didn't receive errors from bungee (apparently other plugin was removing error stacktrace) i didnt notice what was the issue.

Issue was that i was creating the scoreboard packet again (with the same name) so objective already existed. I simply had to send the packet to display it, instead of creating a new one. I need to say that this is the best scoreboard utils i could find.

I always had issues with objectives, teams, etc. But this plugin solved everything, thanks for creating and for your support.
Everything is working fine now. I'm using it in my plugin [AlonsoPvP](https://www.spigotmc.org/resources/84049/). 

I'll use it in all minigames I'll code ^^ Thank you 😄
